restructuredtext_lint/cli.py

```diff
@@ -5,20 +5,36 @@
 import os
 import sys
 
+from collections import OrderedDict
+
+from docutils.utils import Reporter
+
 from restructuredtext_lint.lint import lint_file
 
+# Generate our levels mapping for humans, using ordered dict for --help string
+# http://repo.or.cz/docutils.git/blob/422cede485668203abc01c76ca317578ff634b30:/docutils/docutils/utils/__init__.py#l65
+LEVEL_MAP = OrderedDict([
```

Comment:
It looks like the git commit comment was truncated. I'm using an ordered dict in order that the `` --level``  argparse ``choices`` argument gets the names sorted by severity for display in the help text.